templates/base.html

```diff
@@ -14,10 +14,10 @@
 <body class="text-center">
 {% block nav %}
     {% from "macros.html" import nav_button with context %}
+    <img class="tieto_logo" src="/static/img/logo.png" alt="logo">
```

Comment:
I think we should't use hardcoded static link:
https://stackoverflow.com/questions/16351826/link-to-flask-static-files-with-url-for?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa